sympy/solvers/inequalities.py

```diff
@@ -603,7 +623,11 @@ def valid(x):
             if valid(_pt(start, end)):
                 sol_sets.append(Interval.open(start, end))
 
-            rv = Union(*sol_sets).subs(gen, _gen)
+            if expanded_e.coeff(I) != S.Zero:
+                rv = (make_real)
+
```

Comment:
@akash9712 This extra blank line is not needed.

Reply:
@ArifAhmed1995 I added that line to seperate the if clauses, as done before this pull request. Are you sure it should be removed?

Reply:
Yeah, I think blank line `628` should be removed.